_posts/2018-10-07-how-to-use-elixir-ls-with-vim.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: How to use Elixir LS with Vim
+date: 2018-10-07 09:00:00 -04:00
+categories: post
+desc: Guide on how to use the Elixir LS language server with the Vim text editor. 
+permalink: /:categories/:year/:month/:day/:title/
+---
+
+### What is Elixir LS?
+
+[Elixir LS](https://github.com/JakeBecker/elixir-ls) by Jake Becker is the language server implementation for Elixir.
+
+### What is a language server?
+
+If you've been following the story of Visual Studio Code, there is a chance you've heard of another recent creation from Microsoft: the Language Server Protocol. 
+
+>The Language Server Protocol (LSP) defines the protocol used between an editor or IDE and a language server that provides language features like auto complete, go to definition, find all references etc.
+
+This allows creators to build universal "language servers" that can be used by any text editor. These servers can provide features like code completion, go to definition, and code diagnostics.
```

Comment:
`can provide features like...` is this a restatement of the quote above?

Reply:
I was reading this over last night and I also made that realization. I will probably cut my restatement.